package.json

```diff
@@ -6,10 +6,10 @@
     "scripts": {
         "build": "NODE_ENV=production webpack --config webpack.prod.js && npm run clean-background-prod",
         "dev": "NODE_ENV=development webpack --config webpack.dev.js && npm run clean-background-dev",
-        "clean-background-dev": "source scripts/clean-dev.sh",
```

Comment:
@0xmad Could you please also undo this change? since I renamed the script to `clean-dev.sh` but didn't push it yet. Thanks